config.go

```diff
@@ -93,6 +94,10 @@ func (c *Config) Load() error {
 	}
 	c.Plugins = plugins
 
+	if c.IdFieldName != "" {
```

Comment:
```suggestion
	if strings.TrimSpace(c.IdFieldName) != "" {
```
it's possibly worth being defensive here in case some empty but non blank strings sneak their way into the config.